programmers/스택 큐.같은 숫자는 싫어/just-stopyoon.py

```diff
@@ -0,0 +1,9 @@
+def solution(arr):
+    answer = []
+    answer.append(arr[0])
```

Comment:
```suggestion
    answer = [arr[0]]
```

결국 동일한 의미이지만 저는 이렇게 초기화를 해주어서 이렇게 할 수도 있을 것 같아 남겨둡니다 ㅎㅎ